anax-in-k8s/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:3.6
```

Comment:
We also need a UBI based container, again reference the css dockerfiles for how to do this.